templates/contest/list.html

```diff
@@ -21,7 +21,9 @@
                 count_down($(this));
             });
 
-            $('.contest-tag').find('a').featherlight();
+            if($('.contest-tag').find('a').hasOwnProperty('data-featherlight')) {
```

Comment:
I feel like `.find('a[data-featherlight]')` or similar should be sufficient here.

Reply:
It's also hilarious how wrong that code is.